Dockerfile.migrations

```diff
@@ -18,10 +18,6 @@ RUN cd /tmp/f8a_ingestion/ && \
 COPY alembic.ini hack/run-db-migrations.sh ${ALEMBIC_DIR}/
 COPY alembic/ ${ALEMBIC_DIR}/alembic
 
+RUN chmod 777 ${ALEMBIC_DIR}/run-db-migrations.sh
 
-RUN pip3 install alembic
-
-RUN mkdir -p /f8a_ingestion
-WORKDIR /f8a_ingestion
-
-ENTRYPOINT ["bash", "-c"]
\ No newline at end of file
+ENTRYPOINT ["/alembic/run-db-migrations.sh"]
```

Comment:
I believe you need to install `postgresql` RPM package, otherwise `psql` command won't be available in the image.